grouper/fe/settings.py

```diff
@@ -22,6 +22,7 @@ def override_timezone(self, value):
     "date_format": "%Y-%m-%d %I:%M %p",
     "cdnjs_prefix": "//cdnjs.cloudflare.com",
     "user_auth_header": "X-Grouper-User",
+    "refresh_interval": 60,
```

Comment:
This seems pretty infrequent; does this mean it'll be a full minute for each update when setting up new users?